### PR TITLE
ReactiveStreamsPublisherTckTest - setup cleanupLatch only when starting server

### PR DESCRIPTION
**Pull request by aubi**
The test was waiting for cleanupLatch, although it wasn't always used,
which caused timeouts:
[ReactiveStreamsPublisherTckTest] [ERROR] Server did not close long after test completed

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/ReactiveStreamsPublisherTckTest.java [hunk 1]
```diff
@@ -26,10 +23,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
-
 import javax.ws.rs.sse.InboundSseEvent;
-
 import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import static org.eclipse.microprofile.rest.client.tck.sse.AbstractSseTest.PORT;
+import static org.eclipse.microprofile.rest.client.tck.sse.AbstractSseTest.launchServer;
```
Comment (andymc12) [inline tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/ReactiveStreamsPublisherTckTest.java:29]: Would you undo the import order changes?  My guess is that your IDE did that for you automatically - but there may be a switch to disable that save-action.

**Review by andymc12 — CHANGES_REQUESTED** [review]
Sorry for the delayed review.  I think this looks good, but would you please undo the import changes and also update the copyright?  It should change from:
```
Copyright (c) 2020 Contributors to the Eclipse Foundation
```

to:
```
Copyright (c) 2020, 2021 Contributors to the Eclipse Foundation
```

After that I should be able to merge it.  Thanks!

File: tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/ReactiveStreamsPublisherTckTest.java (hunk 1 above)
Comment (aubi) [reply tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/ReactiveStreamsPublisherTckTest.java]: Exactly as you said. Reverted.

**Review by andymc12 — APPROVED** [review]
Thanks so much!

**Pull request merged**